<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BioStat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;verdana&quot;;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)</string>
    </property>
    <property name="text">
     <string>Добавьте количество продукта (в граммах)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {font: 13pt &quot;verdana&quot;;
background-color: rgb(255, 255, 255);
border-radius: 2px;
border: 2px solid #9a9;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(231, 253, 255, 0.7); border: 2px solid #9a9;}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="singleStep">
     <number>25</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOkCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {font: 13pt &quot;verdana&quot;;
background-color: rgb(255, 255, 255);
border-radius: 2px;
border: 2px solid #9a9;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(231, 253, 255, 0.7); border: 2px solid #9a9;}</string>
    </property>
    <property name="text">
     <string>ОК</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>394</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
